scripts/decode.py

```diff
@@ -5,41 +5,44 @@
 
```

Comment:
Let's just remove this script. It is superceded by the BeamSearchDecoderWithLM

Reply:
Done.